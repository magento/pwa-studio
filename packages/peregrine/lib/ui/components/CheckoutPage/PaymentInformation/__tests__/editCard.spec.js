import React from 'react';
import createTestInstance from '@magento/peregrine/lib/util/createTestInstance';
import EditCard from '../editCard';

jest.mock('../../../../classify');
jest.mock('../creditCard', () => props => <mock-CreditCard {...props} />);

test('test renders correctly', () => {
    const tree = createTestInstance(
        <EditCard
            onPaymentReady={jest.fn()}
            onPaymentSuccess={jest.fn()}
            onPaymentError={jest.fn()}
            resetShouldSubmit={jest.fn()}
            shouldSubmit={false}
        />
    );
    expect(tree.toJSON()).toMatchSnapshot();
});
