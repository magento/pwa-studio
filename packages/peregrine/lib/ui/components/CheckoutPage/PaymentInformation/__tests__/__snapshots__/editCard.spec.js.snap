// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test renders correctly 1`] = `
<div
  className="root"
>
  <mock-CreditCard
    onPaymentError={[MockFunction]}
    onPaymentReady={[MockFunction]}
    onPaymentSuccess={[MockFunction]}
    resetShouldSubmit={[MockFunction]}
    shouldSubmit={false}
  />
</div>
`;
